exchanges/gateio/gateio.go

```diff
@@ -1345,6 +1345,29 @@ func (g *Gateio) GetUsersTotalBalance(ctx context.Context, ccy currency.Code) (*
 	return response, g.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, walletTotalBalanceEPL, http.MethodGet, walletTotalBalance, params, nil, &response)
 }
 
+// ConvertSmallBalances converts small balances of supported crypto (excluding delisted crypto) above 0.00000001 GT and
+// below 0.0001 BTC in value to GT. You can make a conversion every 6 hours. A quota of 100 GT is available per account
+// every 3 days.
+// Providing a list of currency codes will convert only those, else all small currencies will be converted.
+func (g *Gateio) ConvertSmallBalances(ctx context.Context, codes ...currency.Code) error {
```

Comment:
🐒 Appreciate this took time to write.
* However I don't think we should transcribe their current business rules over into our comments, because nearly everything here is likely to change
* Is 6 hours actually correct? The api docs don't mention it, but [this](https://www.gate.io/help/guide/functional_guidelines/22367) says 24 hours.
* Suggest `codes` becomes `currs` or `currencies`
```suggestion
// ConvertSmallBalances converts small balances of provided currencies into GT.
// If no currencies are provided, all supported currencies will be converted
// See [this documentation](https://www.gate.io/help/guide/functional_guidelines/22367) for details and restrictions.
func (g *Gateio) ConvertSmallBalances(ctx context.Context, currs ...currency.Code) error {
```